fragments/infra-boot.sh

```diff
@@ -198,6 +198,11 @@ function clone_openshift_ansible() {
 # =============================================================================
 # MAIN
 # =============================================================================
+function sudo_enable_from_ssh() {
+    sed -i "/requiretty/s/^/#/" /etc/sudoers
+}
+
+sudo_enable_from_ssh
```

Comment:
This will be removed when https://github.com/redhat-openstack/openshift-on-openstack/pull/81 is merged